<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingEditor</class>
 <widget class="QWidget" name="SettingEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="SearchLineEdit" name="search_input"/>
   </item>
   <item row="1" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="styleSheet">
      <string notr="true">QSplitter::handle{
    background-color:rgb(235,235,235);
}</string>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <widget class="TreeView" name="navigation">
      <property name="editTriggers">
       <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
      </property>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="ScrollArea" name="edit_area">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="edit_area_content">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>130</width>
         <height>572</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="edit_layout">
        <item>
         <spacer name="compress_spacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TreeView</class>
   <extends>QTreeView</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>SearchLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ScrollArea</class>
   <extends>QScrollArea</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
